www-servers/fnord/fnord-1.11-r1.ebuild

```diff
@@ -19,28 +19,25 @@ RDEPEND="${DEPEND}
 	virtual/daemontools
 	sys-apps/ucspi-tcp"
 
+DOCS=( TODO README README.auth SPEED CHANGES )
+PATCHES=( "${FILESDIR}/${PN}"-1.10-gentoo.diff )
+
 pkg_setup() {
 	enewgroup nofiles 200
 	enewuser fnord -1 -1 /etc/fnord nofiles
 	enewuser fnordlog -1 -1 /etc/fnord nofiles
 }
 
-src_prepare() {
-	epatch "${FILESDIR}/${PN}"-1.10-gentoo.diff
-}
-
 src_compile() {
 	# Fix for bug #45716
-	replace-sparc64-flags
+	use sparc && replace-sparc64-flags
 
-	use auth && \
-		append-flags -DAUTH
+	use auth && append-flags -DAUTH
 
-	emake DIET="" CC=$(tc-getCC) \
-		CFLAGS="${CFLAGS}" LDFLAGS="${LDFLAGS}" || die "emake failed"
+	emake DIET="" CC=$(tc-getCC) CFLAGS="${CFLAGS}" LDFLAGS="${LDFLAGS}"
 }
 
 src_install () {
 	dobin fnord-conf fnord || die
```

Comment:
dobin needs no || die anymore see 
https://devmanual.gentoo.org/ebuild-writing/eapi/
the rest looks good to me though I can not test it on my system now. I am still busy with gcc updates.